lib/app.js

```diff
@@ -1,17 +1,27 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const cors = require('cors');
 
+// eslint-disable-next-line no-unused-vars
 const authenticate = require('./middleware/authenticate');
 
 const app = express();
 
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(
+  cors({
+    origin:
+      'https://alchemy-shopping-front-end-demo.netlify.app',
+    credentials: true
+  })
+);
 
 // App routes
 
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
just FYI, since the authenticate middleware is required for all the routes in the items controller, you can add that in here instead

`app.use('/api/v1/items', authenticate, require('./controllers/items'));`